src/components/siteButton.tsx

```diff
@@ -95,6 +102,10 @@ const SiteButton: React.FC<ButtonProps> = ({
       //selected state
       [` ${buttonColors[colorScheme].color2} ${buttonColors[colorScheme].color4} ${buttonColors[colorScheme].color7} ${buttonColors[colorScheme].color8}`]:
         isSelected && variant === "hollow",
+
+      //padding
+      "px-8 sm:px-12": padding === "more",
```

Comment:
You shouldn't declare padding in more than one place. It makes it difficult to figure out what padding is getting applied at any give time. Either declare padding in the size classes or remove it from there and declare it here.